### ISSUE TITLE
@documentation/@comment is not supported on enums (assuming it should behave like a Class)

### ISSUE THREAD

**Issue by hoijnet** [terminusdb/terminusdb#774]
**Describe the bug**
Providing an @documentation element is accepted by the schema document interface, but not loaded into the Enum schema element. The assumption is that Enums should also be able to add @comment fields on.

**To Reproduce**
1. Create an Enum document that includes a string in @documentation/@comment
2. Submit the document through the schema document API
3. The Enum schema document is created
4. Request the schema Enum document
5. The schema definition element lacks @documentation/@comment elements

**Expected behavior**
Option a)
If adding @documentation/@comment and it is not supported, reject the document as it is not well-formed
Option b)
Accept @documenation/@comment Enum schema elements
Ensure @documenation/@comment is returned when requesting the Enum schema element

**Screenshots**
Submitted document
```
{
  "@id": "currency",
  "@documentation": { "@comment": "Test" },
  "@type": "Enum",
  "@value": [
    "SEK",
    "USD",
    "EUR"
  ]
}
```

Returned document
```
{
  '@id': 'currency',
  '@type': 'Enum',
  '@value': [ 'SEK', 'USD', 'EUR' ]
}
```

**Info (please complete the following information):**
 - OS: MacOS
 - Using Docker directly - Image terminusdb/terminusdb-server:v10.0.9

**Additional context**
@documentation/@comment works with regular documents and subdocuments, expectation is that they should work the same

**Comment by spl** [comment]
@Francesca-Bit confirmed this in <https://github.com/terminusdb/terminusdb/issues/754#issuecomment-970333431>.

**Comment by spl** [comment]
Fixed by #799.
